<launch>

  <arg name="joy_dev" default="/dev/input/js1" />
  <arg name="joy_autorepeat_rate" default="20" />
  <arg name="joy_deadzone" default="0.05" />

  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="$(arg joy_dev)" />
    <param name="deadzone" value="$(arg joy_deadzone)" />
    <param name="autorepeat_rate" value="$(arg joy_autorepeat_rate)" />
  </node>

  <node pkg="oxar_is_bored" type="teleop_arm_and_wheels.py" name="teleop_node" output="screen">
    <remap from="home_steering" to="base_driver/home_steering" />
  </node>

</launch>
